packages/account/src/providers/provider.test.ts

```diff
@@ -1075,13 +1075,15 @@ Supported fuel-core version: ${mock.supportedVersion}.`
   // TODO: validate if this test still makes sense
   it.skip('should ensure estimated fee values on getTransactionCost are never 0', async () => {
```

Comment:
I think this test should still be run, otherwise if this test is being skipped what's the purpose of these changes?

Reply:
> TODO: validate if this test still makes sense

With the following todo - I am unsure about this test.

From what I can see, the `minFee` will never be `0` as we add `1` after the `calculateGasFee`:
https://github.com/FuelLabs/fuels-ts/blob/8776b79986b8c47b34a9954278c3d9eafd504c09/packages/account/src/providers/provider.ts#L1024-L1029

Do you think this test is still valid?

Reply:
That's correct but I think the test ensures that code remains as such, so I think it's valid. But I don't have the full context.

Reply:
@petertonysmith94 This test can be skipped for now. We can unskip this after https://github.com/FuelLabs/fuels-ts/issues/2813 is solved.

If we cannot use a gas price of `0`, the `maxFee` will never be `0`.

> From what I can see, the minFee will never be 0 as we add 1 after the calculateGasFee:

@petertonysmith94 you are correct, since `calculateGasFee` is always adding `1`, even when the `gasPrice` is `0`, the `maxFee` | `minFee` will never be `0`. 

But this can serve as validation that we are always adding `1` to `maxFee` and `minFee`.

Reply:
@Torres-ssf I think I'll wait for the merge of #2814 - may as well resolve this as one. Unless you'd advise differently?

Reply:
@petertonysmith94 That is ok. We can wait for https://github.com/FuelLabs/fuels-ts/pull/2814

Reply:
Will resolve this after this has been merged:
- https://github.com/FuelLabs/fuels-ts/pull/2814